experimentation/tools/scripts/GetKeyAndDescription.java

```diff
@@ -0,0 +1,51 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Set;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonPrimitive;
```

Comment:
I am not sure about using these packages. I googled how to return a JSON object and got this as the answer. Luckily, these packages were in the classpath.

Reply:
Gson is used in Sorald to produce the stats files.